### ISSUE TITLE
2_populate_raw_tables.py - Freddie performance confused as freddie origination

### ISSUE THREAD

**Issue by JamesPaulKnox** [JamesPaulKnox/mortgage-database#2]
After fixing issue #1, I attempted to populate the db with freddie performance & obligation files. As shown below, Q22000 populated (seemingly) fine. Then, it seems to have attempted to load Q22000 a second time, throwing an error. It tried to load it the second time into origination, which didnt work because it's a performance file.

```
[james@decimator scripts]$ python 2_populate_raw_tables.py 
/run/media/james/MediaB/projects/agency_loan_level/import/freddie_performance/historical_data1_time_Q22000.txt
/run/media/james/MediaB/projects/agency_loan_level/import/freddie_performance/historical_data1_time_Q22000.txt
Traceback (most recent call last):
  File "2_populate_raw_tables.py", line 34, in <module>
    populate("raw_freddie_origination", 26)
  File "2_populate_raw_tables.py", line 18, in populate
    c.execute("INSERT INTO " + table + " VALUES (" + data + ");")
sqlite3.OperationalError: table raw_freddie_origination has 27 columns but 26 values were supplied
```

As I typed the above explanation, it hit me why this happened! This is the code that determines if a given file is fannie origination, fannie performance, freddie origination, or freddie performance.

```
for current in files:
	if "Acquisition_" in current and not "HARP" in current:
		populate("raw_fannie_origination", 25)
	if "Performance_" in current and not "HARP" in current:
		populate("raw_fannie_performance", 31)
	if "historical_data1_time_" in current and not "harp" in current:
		populate("raw_freddie_performance", 26)
	if "historical_data1_" in current and not "harp" in current:
		populate("raw_freddie_origination", 26)
```

(The print statement that displays the file name is within populate())

Here's what I think is happening: 
If statement 1 is false.
If statement 2 is false.
If statement 3 is true (The correct one)
If statement 4 is also true

It was my intention that it would go through the if statements and execute the correct populate arguments, then _go to the next file_ in the list. However, I accidentally assumed it would break the if block once one returned true! That's why it tried to populate the same file into both performance and obligation, because the if statement is true for both.

As I see it, there are two solutions: use elif's instead, or to append a `and not "time_"` to if statement 4. Elif's are probably considered the more pythonic way to go about it. Adding the additional conditional would make the code harder to understand.

**Comment by JamesPaulKnox** [comment]
Fixed in c1b1e74

**Comment by JamesPaulKnox** [comment]
Fixed in c1b1e74
